tools/sigh.ts

```diff
@@ -586,27 +593,38 @@ function spawnWasSuccessful(result: RawSpawnResult, opts: SpawnOptions = {}): bo
   return false;
 }
 
-// make spawn work more or less the same way cross-platform
-function saneSpawn(cmd: string, args: string[], opts?: SpawnOptions): boolean {
+// make spawnSync work more or less the same way cross-platform
+function saneSpawnSync(cmd: string, args: string[], opts?: SpawnOptions): boolean {
   cmd = path.normalize(cmd);
   opts = {stdio: 'inherit', ...opts, shell: true};
   if (opts.logCmd) {
     sighLog('+', cmd, args.join(' '));
   }
   // it's OK, I know what I'm doing
-  const result: RawSpawnResult = _DO_NOT_USE_spawn(cmd, args, opts);
+  const result: RawSpawnResult = _DO_NOT_USE_spawnSync(cmd, args, opts);
   return spawnWasSuccessful(result, opts);
 }
 
 // make spawn work more or less the same way cross-platform
-function saneSpawnWithOutput(cmd: string, args: string[], opts?: SpawnOptions): SpawnResult {
+function saneSpawn(cmd: string, args: string[], opts?: SpawnOptions): ChildProcess {
```

Comment:
How about saneSpawnAsync?

Reply:
I consciously tried to stick to the convention in ChildProcess, as these are just wrappers. I switched only at the top level, because I wanted to keep devServer meaning what it has all along.